lib/ahoy_email.rb

```diff
@@ -29,7 +29,7 @@ module AhoyEmail
     utm_term: nil,
     utm_content: nil,
     utm_campaign: -> { action_name },
-    user: -> { @user || (respond_to?(:params) && params && params[:user]) || (message.to.size == 1 ? (User.find_by(email: message.to.first) rescue nil) : nil) },
+    user: -> { @user || (respond_to?(:params) && params && params[:user]) || (message.to&.size == 1 ? (User.find_by(email: message.to.first) rescue nil) : nil) },
```

Comment:
I'm not sure if all targetted versions support safe navigation operator. Easy enough change if needed